File: scenarios/js/parallel_shell_commands.js
var noOutputColorConfig = {
    "output" : {
        "color" : false
    }
};

var command1 = command
    .config(noOutputColorConfig)
    .value("echo \"Hello world!\"");

var command2 = command
    .value("ps alx | grep java");

parallel
    .step(command1)
    .step(command2)
    .run();

File: scenarios/py/parallel_shell_commands.py
noOutputColorConfig = {
    "output": {
        "color": False
    }
}

command1 = command\
    .config(noOutputColorConfig)\
    .value("echo \"Hello world!\"")

command2 = command\
    .value("ps alx | grep java")

parallel\
    .step(command1)\
    .step(command2)\
    .run()
